### PR TITLE
#3485 - Student: New applications: Student can no longer immediately add trusted contact when "Yes" is selected

### PR DESCRIPTION
**Pull request by andrepestana-aot**
- Added default value to the DataGrid component;
- Removed extra required field asterisks;
- Added MAXIMUM_IDLE_TIME* vars to docker-compose to fix a bug with the local API container.

image

### CODE REVIEW

File: sources/docker-compose.yml [hunk 1]
```diff
@@ -63,6 +63,10 @@ services:
       - S3_ENDPOINT=${S3_ENDPOINT}
       - S3_DEFAULT_BUCKET=${S3_DEFAULT_BUCKET}
       - S3_REGION=${S3_REGION}
+      - MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT=${MAXIMUM_IDLE_TIME_FOR_WARNING_STUDENT}
```
Comment (andrewsignori-aot) [inline sources/docker-compose.yml:66]: Thanks for fixing it ❤️

**Review by andrewsignori-aot — APPROVED** [review]
Great work and solution for the UI. Looks good 👍

File: sources/packages/web/src/assets/css/formio-shared.scss [hunk 2]
```diff
@@ -705,3 +705,13 @@
 .formio-component-file ul li.list-group-item {
   border: 1px solid rgba(0, 0, 0, 0.125) !important;
 }
+
+// Removes the required sign from the input fields with .no-required-field-sign.
+.no-required-field-sign label.field-required:after {
+  content: none;
```
Comment (sh16011993) [inline sources/packages/web/src/assets/css/formio-shared.scss:711]: Should we force here with `!important` as well?

File: sources/packages/forms/src/form-definitions/sfaa2024-25.json [hunk 3]
```diff
@@ -11249,7 +11249,13 @@
                   "disabled": false,
                   "tableView": false,
                   "modalEdit": false,
-                  "defaultValue": [],
+                  "defaultValue": [
```
Comment (sh16011993) [inline sources/packages/forms/src/form-definitions/sfaa2024-25.json:11252]: Can there be at most two trusted contacts only?

**Review by sh16011993 — APPROVED** [review]
Great work @andrepestana-aot  👍  Thanks for the walkthrough.

Left two quick questions (one is more of a business question).

File: sources/packages/forms/src/form-definitions/sfaa2024-25.json (hunk 3 above)
Comment (andrepestana-aot) [reply sources/packages/forms/src/form-definitions/sfaa2024-25.json:11252]: Yes, that was the existing logic.

File: sources/packages/web/src/assets/css/formio-shared.scss (hunk 2 above)
Comment (andrepestana-aot) [reply sources/packages/web/src/assets/css/formio-shared.scss:711]: No need for that one as there wasn't a class preceding and overwriting it.

**Pull request merged**